examples/Image Classification/ios/ImageClassifier.xcodeproj/xcuserdata/mba0077.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,7 @@
 		<key>ImageClassifier.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>3</integer>
+			<integer>4</integer>
```

Comment:
This file can be removed from the project, right? I'm assuming it doesn't need to be committed, and it'd remove the merge conflict.

Reply:
I deleted it